### PR TITLE
January 2024 IaC policies

### PR DESCRIPTION
**Pull request by tsmithv11**

### CODE REVIEW

File: docs/en/enterprise-edition/policy-reference/azure-policies/azure-iam-policies/bc-azure-2-47.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+This policy is identifying whether a storage account has been configured to allow anonymous access to blob data. Anonymous access to blob data means anyone on the internet can retrieve or view data without requiring any form of authentication. This is potentially dangerous because it exposes sensitive data to the public, and it could lead to data leaks and breaches of privacy. Hence, it is critical to ensure storage accounts are not configured with blob anonymous access to safeguard information and maintain data privacy.
+
+=== Fix - Buildtime
+
+*Terraform*
+
+* *Resource:* azurerm_storage_account
+* *Arguments:* allow_nested_items_to_be_public
+
+The problem here is that the Azure storage account is configured with blob anonymous access. Enabling anonymous access to blob content in a storage account allows anyone to read the blob content. This is insecure as sensitive data may be exposed.
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/policy-reference/azure-policies/azure-iam-policies/bc-azure-2-47.adoc:37]: The Azure storage account is configured with anonymous blob access. Enabling anonymous access to blob content in a storage account allows anyone to read the blob content. This is insecure practice, as sensitive data may be exposed.
@tsmithv11 : Why do we need this if there is a description above with the same message?

File: docs/en/enterprise-edition/policy-reference/azure-policies/azure-iam-policies/bc-azure-2-47.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+To fix the issue, ensure that anonymous access to the blob content of your storage account is disabled. You can configure this by setting `allow_nested_items_to_be_public` attribute to `false`.
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/policy-reference/azure-policies/azure-iam-policies/bc-azure-2-47.adoc:39]: To fix the issue, disable anonymous access to the blob content of your storage account: Set the `allow_nested_items_to_be_public` attribute to `false`. This ensures that no one can read blob content without proper authentication and authorization.

File: docs/en/enterprise-edition/policy-reference/azure-policies/azure-iam-policies/bc-azure-2-47.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+[source,hcl]
+----
+resource "azurerm_storage_account" "storage_account" {
+    ...
+    allow_nested_items_to_be_public = false
+}
+----
+
+The code above creates a Azure storage account with `allow_nested_items_to_be_public` set to `false`. Therefore, anonymous access to the blob content is disabled, which will make the storage account secure from unauthorized access. This ensures that no one can read blob content without proper authentication and authorization.
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/policy-reference/azure-policies/azure-iam-policies/bc-azure-2-47.adoc:49]: This code sets the  the `allow_nested_items_to_be_public`' attribute to 'false', disabling anonymous access to the blob content.

File: docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-233.adoc [hunk 4]
```diff
@@ -0,0 +1,53 @@
+
+== Azure Container Registry (ACR) is not zone redundant
```
[the remark this answers is not in the document]
Comment (tsmithv11) [reply docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-233.adoc:2]: 2 works

File: docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-235.adoc [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+|Run,Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy identifies Azure Container Instances (ACI) in which the environment variables with regular value type instead of the secure values property. Objects with secure values are intended to hold sensitive information like passwords or keys for your application. Using secure values for environment variables is both safer and more flexible than including them in your container's image. So it is recommended to secure the environment variable by specifying the 'secureValue' property instead of the regular 'value' for the variable's type.
```
[the remark this answers is not in the document]
Comment (tsmithv11) [reply docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-235.adoc:28]: It is a bit tricky, but I think it's good to leave in this section.

File: docs/en/enterprise-edition/policy-reference/azure-policies/azure-iam-policies/bc-azure-2-47.adoc (hunk 1 above)
Comment (tsmithv11) [reply docs/en/enterprise-edition/policy-reference/azure-policies/azure-iam-policies/bc-azure-2-47.adoc:37]: It's redundant. I'll adjust.

**Comment by JBakstPaloAlto** [comment]
Merging as per @tsmithv11 request

**Pull request merged**